From 202a21eb98e117dbfcb9bfce88febd408f2742cd Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Date: Fri, 15 Mar 2013 17:08:40 +0100
Subject: Disable Firefox Health Report


diff --git a/browser/confvars.sh b/browser/confvars.sh
index fab0d13..9956360 100755
--- a/browser/confvars.sh
+++ b/browser/confvars.sh
@@ -28,7 +28,7 @@ MOZ_DISABLE_EXPORT_JS=1
 MOZ_SAFE_BROWSING=1
 MOZ_SERVICES_COMMON=1
 MOZ_SERVICES_CRYPTO=1
-MOZ_SERVICES_HEALTHREPORT=1
+MOZ_SERVICES_HEALTHREPORT=
 MOZ_SERVICES_METRICS=1
 MOZ_SERVICES_SYNC=1
 MOZ_SERVICES_CLOUDSYNC=1
-- 
cgit v0.10.2
